packages/react-aria-components/docs/examples/command-palette.mdx

```diff
@@ -0,0 +1,147 @@
+{/* Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {ExampleLayout} from '@react-spectrum/docs';
+export default ExampleLayout;
+
+import docs from 'docs:react-aria-components';
+import styles from '@react-spectrum/docs/src/docs.css';
+import Menu from '@react-spectrum/docs/pages/assets/component-illustrations/Menu.svg';
+import TextField from '@react-spectrum/docs/pages/assets/component-illustrations/TextField.svg';
+import {ExampleCard} from '@react-spectrum/docs/src/ExampleCard';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+
+---
+keywords: [example, autocomplete, menu, aria, accessibility, react, component]
+type: component
+image: command-palette.png
+description: An command palette with actions, styled with Tailwind CSS.
+---
+
+# Command Palette
+
+A Command Palette is an interface that allows users to quickly run commands or navigate to content within an application. This example uses the [`Autocomplete`](../Autocomplete.html) component from React Aria Components to filter a list of commands and display them in a [`Menu`](../Menu.html). The [`TextField`](../TextField.html) is used to capture user input and filter the list of available commands.
+
+```tsx import
+import './tailwind.global.css';
+```
+
+```tsx example standalone
+import {
+  UNSTABLE_Autocomplete as Autocomplete,
+  TextField,
+  Menu,
+  MenuItem,
+  useFilter,
+  Label,
+  Input,
+  Button
+} from 'react-aria-components';
+
+function CommandPaletteExample() {
+  let commands = [
+    {id: 'new-file', label: 'Create new file…'},
+    {id: 'new-folder', label: 'Create new folder…'},
+    {id: 'assign', label: 'Assign to…'},
+    {id: 'assign-me', label: 'Assign to me'},
+    {id: 'status', label: 'Change status…'},
+    {id: 'priority', label: 'Change priority…'},
+    {id: 'label-add', label: 'Add label…'},
+    {id: 'label-remove', label: 'Remove label…'}
+  ];
+
+  let {contains} = useFilter({sensitivity: 'base'});
+
+  return (
+    <div className="bg-linear-to-r from-indigo-500 to-violet-500 p-8 h-[340px] rounded-lg flex flex-col items-center justify-center">
+      <div className="flex flex-col gap-1 w-[320px] rounded-md bg-white shadow-lg">
```

Comment:
Do we want to make this into a Dialog that you could open with Command K (or by clicking a button)?